lib/ansible/modules/cloud/amazon/ecs_taskdefinition.py

```diff
@@ -234,9 +268,16 @@ def main():
         containers=dict(required=False, type='list'),
         network_mode=dict(required=False, default='bridge', choices=['bridge', 'host', 'none', 'awsvpc'], type='str'),
         task_role_arn=dict(required=False, default='', type='str'),
-        volumes=dict(required=False, type='list')))
+        volumes=dict(required=False, type='list'),
+        launch_type=dict(required=False, choices=['EC2', 'FARGATE']),
+        cpu=dict(),
+        memory=dict(required=False, type='str')
```

Comment:
Does this need to be an int? The docs https://boto3.readthedocs.io/en/latest/reference/services/ecs.html#ECS.Client.register_task_definition indicate that